x/distribution/src/errors.rs

```diff
@@ -17,3 +27,49 @@ pub enum TokenAllocationError {
     #[error("{0}")]
     BankSend(#[from] BankKeeperError),
 }
+
+#[derive(Debug, Clone, thiserror::Error)]
+pub enum DistributionError {
```

Comment:
Um... are you sure that it should be single error?

Reply:
90% of errors I added following single method that withdraw rewards.  I've added only single error from the other method.

Reply:
Wow